README.md

```diff
@@ -23,6 +23,14 @@ To configure the GitHub webhook, point it to the `/github-hook` path of your
 webserver (by default `http://localhost:8000`), configure the secret you chose
 in `.env`, set the content type to `application/json` and select all events.
 
+## Notes about Github issues/pulls request APIs
+
+The Github API are a bit confusing when the intent is to search either through issues _or_ pull requests. The `/{repo}/pulls` endpoint does not support filter by label, while the `/{repo}/issues` does.
```

Comment:
Are you sure pulls doesn't support labels?, quickly looking at our code it seems like we are using labels for PRs queries.